bindata/network/multus/multus.yaml

```diff
@@ -455,3 +455,39 @@ spec:
           configMap:
             name: cni-binary-copy-script
             defaultMode: 0744
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: ip-reconciler
+  namespace: openshift-multus
+  labels:
+    tier: node
+    app: whereabouts
+spec:
+  # reconcile loop every 5 minutes, starting at the top of the hour
+  schedule: "*/5 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          priorityClassName: "system-cluster-critical"
```

Comment:
Did you actually find any documentation about this field and how to pick a value for it? (We should probably link to that from the wiki so that next time we add something we know where to look...)

Reply:
Unfortunately I didn't. 

I went with the intermediate value. Having said that, I agree that it would be very helpful for the next time it is needed.